macs2.wdl

```diff
@@ -30,8 +30,15 @@ task PeakCalling {
         String sampleName
         String format = "AUTO"
         Boolean nomodel = false
+        String gensz = "hs"
+        Int extsize
+        Int shiftsize = -1*round(extsize/2)
+        Float pval_thres = 0.01
+        Boolean bdg = true
+        String keepdup = "auto"
```

Comment:
So all these valuens are now set by default, isn't it better to make all these optional? So the user can provide their own inputs if wanted?

Reply:
All made optional